.github/workflows/search-api-ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.8,3.9]
```

Comment:
I think it might not have 3.11 yet as that was just released recently

Reply:
also I think you need to add this to the 'testing' flow as well because it seems to be going off of a default version

Reply:
3.12 is the "newest" so 3.11 is there.
Do I need to change testing as part of this? Nothing in that part of the script has changed.

Reply:
I removed 3.11

Reply:
bah this is annoying its not recognizing 3.10 --- Maybe it will work if you change that to 3.10.8?

Reply:
lint passes now, test still chooses wrong version.
will fix that in a different tix